source/administration/production-checklist-operations.txt

```diff
@@ -192,7 +192,7 @@ Linux
    - Configure sufficient file handles (``fs.file-max``), kernel pid limit
      (``kernel.pid_max``), and maximum threads per process
      (``kernel.threads-max``) for your deployment. For large systems,
-     values of 98000, 32768, and 64000 are a good starting point.
+     values of 98000, 64000, and 64000 are a good starting point.
```

Comment:
Ah, I see now.  I didn't realize that the three values listed corresponded to the  file-max, pid_max and threads-max : (